.dev/systemimage/climate_machine_image.jl

```diff
@@ -11,28 +11,26 @@ sysimage_path =
 
 using Pkg
 Pkg.add("PackageCompiler")
+
 using PackageCompiler
 Pkg.activate(joinpath(@__DIR__, "..", ".."))
 Pkg.instantiate()
 pkgs = Pkg.installed()
```

Comment:
When we go to Julia 1.4 we will want to use [`dependencies`](https://julialang.github.io/Pkg.jl/dev/api/#Pkg.dependencies) here since [`installed` is deprecated](https://github.com/JuliaLang/Pkg.jl/blob/127412cd23224a4f8fcdb049d80c4c16486a1e93/src/Pkg.jl#L531-L541)

Reply:
See @simonbyrne already caught this.